src/extras/usdImagingLite/engine.h

```diff
@@ -0,0 +1,188 @@
+/**********************************************************************
+* Copyright 2020 Advanced Micro Devices, Inc
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+********************************************************************/
```

Comment:
I think this needs to include the original license since it's edited from that.

Reply:
Do we need to use original license for all usdImagingLite project files?

Reply:
Files that are copied or edited from Pixar USD need their license.  New ones need ours

Reply:
ok, will do